sdk/ml/azure-ai-ml/azure/ai/ml/operations/_job_operations.py

```diff
@@ -403,8 +403,8 @@ def begin_cancel(self, name: str, **kwargs: Any) -> LROPoller[None]:
         .. admonition:: Example:
 
             .. literalinclude:: ../samples/ml_samples_misc.py
-                :start-after: [START job_operations_begin_cancel]
-                :end-before: [END job_operations_begin_cancel]
+                :start-after: [START job_operations_cancel]
```

Comment:
Why is this being changed? The function name is `begin_cancel` so it's unclear to me why we're removing the `begin` from this.

Reply:
Good catch - that shouldn't have been changed!

Also troubling that Sphinx doesn't catch that...

Reply:
Ah - actually upon looking further this was right. The person who added the example code in samples/ that's referenced here, named it `job_operations_cancel` instead of `job_operations_begin_cancel` but wrote `job_operations_begin_cancel`, so the sample was referring to an example that actually didn't exist. I just changed the name here to match the name in the sample. To make it clearer though, I'll just change the name of the sample to `job_operations_begin_cancel` and match it here.